Detectors/PHOS/testsimulation/drawPHOSgeometry.C

```diff
@@ -86,8 +86,8 @@ void drawPHOSgeometry()
 
   gGeoManager->GetListOfVolumes()->ls();
   gGeoManager->CloseGeometry();
-//gGeoManager->SetVisLevel(0); // (default) TGeo сам решает, сколько уровней геометрии рисовать
-gGeoManager->SetVisLevel(5); // ты сам решает, что нужно нарисовать геометрию с глубиной только до 5 уровней
+gGeoManager->SetVisLevel(0); // (default) TGeo сам решает, сколько уровней геометрии рисовать
+//gGeoManager->SetVisLevel(5); // ты сам решает, что нужно нарисовать геометрию с глубиной только до 5 уровней
```

Comment:
please change the comments to English, again, this looks like accidentally changed